src/main/java/appeng/parts/automation/PlaneModels.java

```diff
@@ -42,34 +43,34 @@
 	public static final ResourceLocation MODEL_CHASSIS_ON = new ResourceLocation( AppEng.MOD_ID, "part/transition_plane_on" );
 	public static final ResourceLocation MODEL_CHASSIS_HAS_CHANNEL = new ResourceLocation( AppEng.MOD_ID, "part/transition_plane_has_channel" );
 
-	private final Map<PlaneConnections, List<ResourceLocation>> modelsOff;
+	private final Map<PlaneConnections, IPartModel> modelsOff;
 
-	private final Map<PlaneConnections, List<ResourceLocation>> modelsOn;
+	private final Map<PlaneConnections, IPartModel> modelsOn;
 
-	private final Map<PlaneConnections, List<ResourceLocation>> modelsHasChannel;
+	private final Map<PlaneConnections, IPartModel> modelsHasChannel;
 
 	public PlaneModels( String prefixOff, String prefixOn )
 	{
-		Map<PlaneConnections, List<ResourceLocation>> modelsOff = new HashMap<>();
-		Map<PlaneConnections, List<ResourceLocation>> modelsOn = new HashMap<>();
-		Map<PlaneConnections, List<ResourceLocation>> modelsHasChannel = new HashMap<>();
+		Map<PlaneConnections, IPartModel> modelsOff = new HashMap<>();
+		Map<PlaneConnections, IPartModel> modelsOn = new HashMap<>();
+		Map<PlaneConnections, IPartModel> modelsHasChannel = new HashMap<>();
 
 		for( PlaneConnections permutation : PlaneConnections.PERMUTATIONS )
 		{
 			ResourceLocation planeOff = new ResourceLocation( AppEng.MOD_ID, prefixOff + permutation.getFilenameSuffix() );
 			ResourceLocation planeOn = new ResourceLocation( AppEng.MOD_ID, prefixOn + permutation.getFilenameSuffix() );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method new appeng.parts.automation.PlaneModels(String, String) appears to call the same method on the same object redundantly [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=fb-contrib%3APRMC_POSSIBLY_REDUNDANT_METHOD_CALLS)